manifests/0000_90_cluster-monitoring-operator_01-dashboards.yaml

```diff
@@ -1868,10 +1868,10 @@ metadata:
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
   labels:
-    app.kubernetes.io/component: grafana
-    app.kubernetes.io/name: grafana
+    app.kubernetes.io/component: dashboard
+    app.kubernetes.io/name: dashboard
     app.kubernetes.io/part-of: openshift-monitoring
-    app.kubernetes.io/version: 8.3.4
+    app.kubernetes.io/version: 7.5.10
```

Comment:
Since removed grafana from [versions.yaml](https://github.com/openshift/cluster-monitoring-operator/pull/1731/files#diff-17f3babfb6ece0926e1825f3209ac9c0812e6b69dd94d3a9bfbdfe0eda735f6c) and updated dashboard.libsonnet (previously named grafana.libsonnet) to use [kubernetes-grafana](https://github.com/brancz/kubernetes-grafana/blob/master/grafana/grafana.libsonnet) instead of [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus/blob/main/jsonnet/kube-prometheus/components/grafana.libsonnet) it is using default version. Since we no longer deploy grafana this version doesn't make sense.

So either we should update to use console version or remove this label

@simonpasquier wdyt?

Reply:
I think it better to use console version as version of dashboards definitions.

Reply:
Hmm, I think we can remove the label. The relevant version would come from kubernetes-monitoring/kubernetes-mixin but it has no tags...